### ISSUE TITLE
[Feature Request] Maintain first party Type GraphQL libraries with your fork

### ISSUE THREAD

**Issue by wSedlacek** [nestjs/graphql#827]
## I'm submitting a...
<pre><code>
[ ] Regression
[ ] Bug report
[x] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
`typegraphql-prisma` is not supported by `@nestjs/graphql` and there is no first party solution provided by `@nestjs/graphql` nor `type-graphql` for support of this library.

## Expected behavior
I would expect `typegraphql-prisma` either be supported or `@nestjs/graphql` to provide an equal alternative.

## Minimal reproduction of the problem with instructions
Attempt to use `typegraphql-prisma` in a project with `@nestjs/graphql` and `@nestjs/graphql` will not see the resolvers generated by `typegraphql-prisma` because a different `ExternalContextCreator` is used.

## What is the motivation / use case for changing the behavior?
In NestJS v7 `type-graphql` was forked into `@nestjs/graphql` with many breaking changes including changing the names of exports and of course changing of the package itself. This is fine for use of `type-graphql` alone and in fact I even prefer the name of the import. The problem comes in with first party libraries that use `type-graphql` such as `typegraphql-prisma`

There has been a community fork to maintain a version of `typegraphql-prisma` for `@nestjs/graphql` (`typegraphql-prisma-nestjs`) however it is maintained by a single person and not updated as quickly as the upstream `typegraphql-prisma` and in my attempts to contribute to it I have found that many of the testing done for `typegraphql-prisma` is broken with the fork.

`type-graphql` itself has shown no intent of maintain support for the fork of `@nestjs/graphql` going as far as to make `typegraphql-nestjs` which completely circumvents `@nestjs/graphql` with a different API and no support for guards, interceptors, filters and pipes. I am sure you as much as me would have liked if `type-graphql` just worked with you to build a single API for nestjs however since they didn't I believe since `@nestjs/graphql` forked from `type-graphql` it is the responsibility of `@nestjs/graphql` to maintain support for any first party libraries of `type-graphql` that are still applicable such as `typegraphql-prisma`.

tl;dr Please maintain a fork of `typegraphql-prisma` for `@nestjs/graphql` if you are going to fork from `type-graphql`

Sources:
- https://github.com/MichalLytek/type-graphql/issues/135
- https://github.com/MichalLytek/type-graphql/issues/476
- https://github.com/EndyKaufman/typegraphql-prisma-nestjs

## Environment
<pre><code>
Nest version: 7.3.4
</code></pre>

**Comment by kamilmysliwiec** [comment]
>  it is the responsibility of @nestjs/graphql to maintain support for any first party libraries of type-graphql that are still applicable such as typegraphql-prisma

`typegraphql-prisma` was released almost at the same time as the new version of this package. I don't think that this means that we are now obliged to support every single feature & integration that `type-graphql` introduces on their side in the future.

There are no plans to integrate `typegraphql-prisma` at the moment.

**Comment by kamilmysliwiec** [comment]
BTW this library is not "a fork". It's almost impossible for us to follow everything that will be added to `type-graphql` so expect that these libraries (and their APIs) will very likely diverse even more in the future.
